plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -335,15 +390,15 @@ protected void buildImage(Dockerfile dockerfile,
         }
     }
 
-    private void pullImage(DockerInstanceKey dockerInstanceKey, final LineConsumer creationLogsOutput) throws MachineException {
-        if (dockerInstanceKey.getRepository() == null) {
+    private void pullImage(DockerMachineSource dockerMachineSource, final LineConsumer creationLogsOutput) throws MachineException {
+        if (dockerMachineSource.getRepository() == null) {
             throw new MachineException("Machine creation failed. Snapshot state is invalid. Please, contact support.");
         }
         try {
             final ProgressLineFormatterImpl progressLineFormatter = new ProgressLineFormatterImpl();
-            docker.pull(dockerInstanceKey.getRepository(),
```

Comment:
it's funny because here https://github.com/eclipse/che/commit/e868d806498a806f3f4213922d8727e593abe230 (same class just one day before mine) you didn't say to not use anymore deprecated methods. So please keep the same review rules for everyone.